src/components/HeaderBar.js

```diff
@@ -46,39 +49,55 @@ class HeaderBar extends React.Component {
         window.removeEventListener( "resize", this.updateBarOpen );
         document.removeEventListener( 'click', this.handleClickOutside );
     }
+    
+
+    handleDarkModeChanged(){
+        if (this.state.darkMode === false) {
+            this.setState( { darkMode : true } )
+        } else {
+            this.setState( { darkMode : false } )
+        }
+        this.props.callbackParent(this.state.darkMode);
+    }
+
 
     render() {
+
         const items = (
-            <div>
+            <div className={this.state.darkMode ? "" : "dark-mode-Headerbar"} >
```

Comment:
just a suggestion
className={ `headerbar${ this.state.darkMode? ' dark-mode' : '' }`}

Reply:
Some places with many class names have been changed.

Reply:
but not all?

Reply:
changed all